docs-rtd/howto/manage-migrations.md

```diff
@@ -0,0 +1,101 @@
+(manage-migrations)=
+
+# Manage migrations
+
+The following document shows you what changes, if any, are needed to your Terraform plan when migrating a model(s) between Juju controllers and/or JAAS.
+
+## Migrating to another Juju controller
+
+When a model is migrated to a new Juju controller, no changes are needed. Simply update you plan to connect to the new controller.
```

Comment:
```suggestion
When a model is migrated to a new Juju controller, no changes are needed. Simply update your plan to connect to the new controller.
```

Reply:
The migration might still fail if the new controller doesn't have the cloud that the model is using or doesn't know about the owner of the model you're migrating.

Reply:
The process of actually performing a model migration is not discussed here so this is all assuming the migration has been completed.